bin/before_install

```diff
@@ -18,3 +18,26 @@ elif [ ! -d "$spec_manageiq" ]; then
   echo "== Cloning manageiq sample app =="
   git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 "$spec_manageiq"
 fi
+
+if [ -n "$CI" ]; then
+  # Gemfile.lock.release only applies to non-master branches and PRs to non-master branches
+  if [[ "$GITHUB_REPOSITORY_OWNER" = "ManageIQ" && "$GITHUB_BASE_REF" != "master" && "$GITHUB_REF_NAME" != "master" && "$GITHUB_REF_NAME" != "dependabot/"* ]]; then
+    echo "== Setup Gemfile.lock.release =="
+    cp -f "$spec_manageiq/Gemfile.lock.release" "$gem_root/Gemfile.lock"
+
+    prog=$(cat << EOF
+import os
+import re
+f = open('Gemfile.lock')
+gemfile_lock = f.read()
+f.close()
+gemfile_lock_patched = re.compile('GIT\n  remote: https://github.com/ManageIQ/manageiq-providers-autosde\n  revision: .+\n  branch: .+\n').sub('PATH\n  remote: .\n', gemfile_lock)
+f = open('Gemfile.lock', 'w')
+f.write(gemfile_lock_patched)
+f.close()
+EOF
+)
+    python3 -c "$prog"
```

Comment:
Once this goes green, I'm wondering if the "right" approach is to store this bit of the script in the core repo.  The reason is that at this point we now have the core repo cloned, and then we can start sharing from the core repo.  This way we don't copy-paste across all of the repos.